lua/terrortown/entities/roles/sidekick/shared.lua

```diff
@@ -49,8 +54,14 @@ hook.Add("TTTUlxDynamicRCVars", "TTTUlxDynamicSikiCVars", function(tbl)
 
 	table.insert(tbl[ROLE_SIDEKICK], {
 		cvar = "ttt2_siki_mode",
-		checkbox = true,
-		desc = "Normal mode for the Sidekick (Def. 1). 1 = Sidekick -> Jackal. 2 = Sidekick receive targets"
+		combobox = true,
+		desc = "Sidekick-Mode (Def. 1)",
+		choices = {
+		"0 = Sidekick doesn't become his former teammate and can't win alone, but gets targets",
+		"1 = Sidekick becomes his former teammate upon their death.",
+		"2 = Sidekick doesn't become his former teammate but can win alone and gets targets"
+		},

```

Comment:
Is that tested? Because Lua starts counting at `1` instead of `0`...

Reply:
It's tested. The convar is a read as a bool in the current version, so the modes start with `0` to make it backwards compatibile.

Reply:
Yea but where do you set the value connected with those choices? Or how can 2 be converted into a bool. I can't find a code snippet of that

Reply:
I just check if the value is `0`, `1`, `>0`, or similar. E.g. line 25 or line 252.

Reply:
Yea that's why I'm asking whether you've tested it and whether it's working, because so far I can't see any connection of that. You creating choices index by `0`, but what if the list's count starts at `1`? Then the wrong convar number is assigned.

Reply:
Choices do start at 0, I implemented it myself. Don't ask me why: gmod shenanigans.

Reply:
uff but yea, ah ok, that's all I needed to aprove this :D